ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,21 +84,80 @@ development team of the Linux Project
 
 - Security Standards
 - License Compliance Policies
-- Tooling
-  - Build in-house
-  - Adopt existing tools
-      - Commercial tools
-      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
-- Design OSS infrastructure
-
-### OS Community Education
 
-#### Create Internal OS Ambassadors
 
-- Drive event sponsorship
-- Train speakers for coding forums
-  
-#### Create educational programs
+### 📝 Establish and Improve Open Source Policies and Processess
+
+##### Policies
+
+- Using and compliance policy
+  - Forking an open source project
+  - Copying open source licensed files/snippets into organization's codebase
+  - Mixing code (owned and third-party OSS/proprietary) and distributing as products or services (sdk, REST API, SaaS, etc.)
+  - Using standalone open source tools (e.g. Firefox, MySQL, CentOS)
+- Policy on open source licenses
```

Comment:
I am not sure what this node is about. Is it about creating and maintaining the OSS policy? If yes, maybe a better name would be: "Create and maintain a policy on open source licenses"? 

Then under this could be:

- Create a white list of OSS licenses
- Create a black list of OSS licenses
- Create a gray list of OSS licenses

Reply:
@winterrocks   

I added /changed  'policies'

- Policy on open source and proprietary licenses
  - Concluding of obligations of licenses
  - Classification of licenses
  - Complying with license obligations

and under the 'Processes' section for the 'Using Open Source' I added a new point  'Maintaining a classification and enforcing rules for open source and proprietary licenses'

P.S. We don't use white/black list anymore but prefer to use allow/deny list so we are more inclusive.